documentation/home-page.asciidoc

```diff
@@ -18,14 +18,65 @@ toc::[]
 
 This is the main page that you will find after your profile creation and the page where you will start from henceforth.
 
-It contains an small introduction, a button to download the latest version and a counter of your projects.
+It contains three sections: +
 
-Click on *Download latest version* to start creating projects.
+    1. Toolbar +
+    2. Sidebar +
+    3. Content +
 
-== Topbar
+== Topbar +
+This section is at the top of the page, it contains *Devon Instance* dropdown to select devonfw-ide that can be used as a base for the projects.
 
-In the bar at the bottom of the page, there is a selector where you can use the devonfw-ide that you want to use as base for your projects.
+image::images/home_page/toolbar.png["Toolbar", width="600", link="images/home_page/toolbar.png"]
 
-== Sidebar
+Next to the *Devon Instance* dropdown, there is a *quick help* icon, clicking on it will open a popup which gives some tips for how to use Devon Dashboard IDE.
```

Comment:
Please, fix all the **devonfw** names in this PR and in other documentation files. Please, remember that it is not Devon, devon, etc. it is **devonfw** in lowercase. Thanks!

Reply:
Hi @sjimenez77,

I updated the changes. Please check it.

Thanks.

Reply:
Ok, thanks.